tests/ltp/ext2fs_tests_unhandled_syscalls.txt

```diff
@@ -1,368 +1,368 @@
-/ltp/testcases/kernel/syscalls/add_key/add_key01: *** kernel panic: syscall.c(5019): myst_syscall(): unhandled syscall: SYS_add_key()
```

Comment:
If we can get rid of the line# in the error message, we wouldn't have so many false updates.

Reply:
Yeah agree. Have changed the local test runner script to log only the syscall name. Will update this file in the next commit.

Reply:
Unhandled syscall list is updated now.